artview/components/tools.py

```diff
@@ -110,6 +110,13 @@ def onPick(self, event):
         ydata = event.ydata # get event y location
         if (xdata is None) or (ydata is None):
             self.msg = "Please choose point inside plot area"
+        elif self.Vradar.value is None:
+            az = np.arctan2(xdata, ydata)*180./np.pi
+            if az < 0:
+                az = az + 360.
+            rng = np.sqrt(xdata*xdata + ydata*ydata)
+            self.msg = 'x = %4.2f, y = %4.2f, Azimuth = %4.2f deg., Range = %4.3f km'\
+                        %(xdata, ydata, az, rng)
```

Comment:
if radar is none, there is no data, but x,y still exist and azi and range as polar coordinates also do so I decided to print that instead of just giving a warning. In else you will get the azi, rng from the actual bean and the value.